RSSReader/View/FeedCell.m

```diff
@@ -0,0 +1,99 @@
+//
+//  FeedCell.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import "FeedCell.h"
+#import "FeedItem.h"
+
+@interface FeedCell ()
+
+@property (nonatomic, retain) FeedItem *feedItem;
+@property (nonatomic, retain, readwrite) UILabel *title;
+@property (nonatomic, retain, readwrite) UILabel *pubDate;
+@property (nonatomic, retain) UIStackView *stackView;
+
+@end
+
+int const kCellSpacing = 10;
+
+@implementation FeedCell
+
+- (void)configureWithItem:(FeedItem *)item {
+    self.feedItem = item;
+}
```

Comment:
right now this method has no differences with the simple setter and it looks too redundant 
I suppose if this is the configuration method, make it do what it intended for - to configure this cell
set text properties on labels and more what you need